app/views/payments/index.html.erb

```diff
@@ -0,0 +1,55 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Payments</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body class="hidden-scroll">
+    <%= link_to expenses_path, class: 'bk' do %>
+      <%= image_tag("back-arrow.png", alt: "Back") %>
+    <% end %>
+  <% sum = 0 %>
+  <div class="payments flex-column-centered hidden-scroll ">
+  <div class="flex-column-centered payment-details hidden-scroll relative">
+  <br/>
+  <br/>
+  <br/>
+  <br/>
+  <br/>
+  <% @payments.each do |payment| %>
+    <% if (payment.transaction_id == @expense.id) %>
+    <% sum += payment.amount %>
+        <div class="detail">
+            <p class="payment-name flex-column-start">
+              <%= payment.name %>
+            </p>
+          <div class="flex-column-start gap-10p relative">
+            <p class="payment-date  flex-start">
+              <p><%= payment.created_at.strftime('%e %B %Y ') %></p>
+              <p><%= payment.created_at.strftime('at %l:%M %p') %></p>
+            </p>
+            <p class="payment-amount  flex-start">
+                 $<%= payment.amount %>
+            </p>
+          </div>
+        </div>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] @kalbek great work on your payments details page however the first payment is not fully visible, maybe you can add a top margin to give it more space from the total section 
![image](https://github.com/kalbek/Budget-app/assets/61408860/5c986761-06cd-4c00-92fc-8ecce38fbc65)